subjects/issorted/README.md

```diff
@@ -16,7 +16,7 @@ func IsSorted(f func(a, b int) int, a []int) bool {
 }
 ```
 
-### Usage
+### Testing
```

Comment:
All of our READMEs have the notation `### Usage`, besides this everything looks good.